src/dmd/astcodegen.d

```diff
@@ -44,6 +44,7 @@ struct ASTCodegen
     alias MODFlags                  = dmd.mtype.MODFlags;
     alias Type                      = dmd.mtype.Type;
     alias Parameter                 = dmd.mtype.Parameter;
+    alias Tarray                    = dmd.mtype.Tarray;
```

Comment:
Not sure why there was no `array` here and why DMD/LDC are fine with it, but GDC doesn't like it:

https://semaphoreci.com/dlang/dmd-2/branches/pull-request-11575/builds/10